packages/rollup-core/src/app/data/producers/l1-chain-data-persister.ts

```diff
@@ -195,15 +195,19 @@ export class L1ChainDataPersister extends ChainDataProcessor {
    * @param blockHash The block hash for the block in which we're searching for logs.
    * @returns The combined array of Logs that we care about based on our topics.
    */
-  private async getLogsForBlock(blockHash: string): Promise<Log[]> {
+  private async getLogsForBlock(block: Block): Promise<Log[]> {
     if (!this.topics.length) {
       return []
     }
 
+    // TODO: Protect against reorgs which change the block number from under us
+    //       Once Ganache supports EIP-234 then we should move this back to using blockHash
+    //       https://github.com/trufflesuite/ganache-core/issues/136
```

Comment:
PR open here to fix this: https://github.com/trufflesuite/ganache-core/pull/635

Reply:
To fix this issue, clone https://github.com/ethereum-optimism/ganache-cli and then run `$ npm run docker` so that it builds the container, exit the CLI and then `$ docker push`. Then we will need to update the `l1_chain.image` in the `docker-compose` config